pyproject.toml

```diff
@@ -21,8 +21,8 @@ classifiers = [
 dynamic = ["version"]
 dependencies = [
     "click>=7.0",
-    "rasterio>=1.3.3",
-    "morecantile>=5.0,<6.0",
+    "rasterio>=1.4.1",
```

Comment:
why do we need this?

Reply:
oh good point... late night PRs are not always a good idea

Reply:
no worries